### ISSUE TITLE
Run podsandbox panic

### ISSUE THREAD

**Issue by Crazykev** [kubernetes/frakti#91]
```
I0315 20:12:03.360743   12416 manager.go:107] RunPodSandbox with request &RunPodSandboxRequest{Config:&PodSandboxConfig{Metadata:&PodSandboxMetadata{Name:busybox,Uid:c57bea76-08a6-11e7-a282-4ccc6a36dc71,Namespace:default,Attempt:0,},Hostname:busybox,LogDirectory:/var/log/pods/c57bea76-08a6-11e7-a282-4ccc6a36dc71,DnsConfig:&DNSConfig{Servers:[8.8.8.8],Searches:[],Options:[ndots:5],},PortMappings:[&PortMapping{Protocol:TCP,ContainerPort:8080,HostPort:0,HostIp:,}],Labels:map[string]string{io.kubernetes.pod.name: busybox,io.kubernetes.pod.namespace: default,io.kubernetes.pod.uid: c57bea76-08a6-11e7-a282-4ccc6a36dc71,name: busybox,},Annotations:map[string]string{kubernetes.io/config.seen: 2017-03-14T19:10:02.897218293+08:00,kubernetes.io/config.source: api,},Linux:&LinuxPodSandboxConfig{CgroupParent:/kubepods/besteffort/podc57bea76-08a6-11e7-a282-4ccc6a36dc71,SecurityContext:&LinuxSandboxSecurityContext{NamespaceOptions:&NamespaceOption{HostNetwork:false,HostPid:false,HostIpc:false,},SelinuxOptions:nil,RunAsUser:nil,ReadonlyRootfs:false,SupplementalGroups:[],Privileged:false,},},},}
I0315 20:12:03.363198   12416 network.go:66] Get interface information in container ns, skip non-veth device lo
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x48b8d2]

goroutine 31 [running]:
panic(0x15fb640, 0xc4200140b0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
k8s.io/frakti/pkg/hyper.networkInfoFromNs(0x2320f00, 0xc420478370, 0x37)
	/home/crazykev/go-project/src/k8s.io/frakti/pkg/hyper/network.go:46 +0xa2
k8s.io/frakti/pkg/hyper.(*Runtime).RunPodSandbox(0xc4203a5b40, 0xc4203b22a0, 0x1d, 0xc42053da40, 0x1, 0x1)
	/home/crazykev/go-project/src/k8s.io/frakti/pkg/hyper/sandbox.go:62 +0x49b
k8s.io/frakti/pkg/manager.(*FraktiManager).RunPodSandbox(0xc4203a5c40, 0x7f4f39ec7000, 0xc4202bacc0, 0xc42008c260, 0x0, 0x1d8, 0x1d8)
	/home/crazykev/go-project/src/k8s.io/frakti/pkg/manager/manager.go:109 +0x123
k8s.io/frakti/vendor/k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime._RuntimeService_RunPodSandbox_Handler(0x176a820, 0xc4203a5c40, 0x7f4f39ec7000, 0xc4202bacc0, 0xc42022d810, 0x0, 0x0, 0x0, 0x523f10, 0xc4204a0070)
	/home/crazykev/go-project/src/k8s.io/frakti/vendor/k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime/api.pb.go:2300 +0x27d
k8s.io/frakti/vendor/google.golang.org/grpc.(*Server).processUnaryRPC(0xc420418f00, 0x2322ee0, 0xc420490a20, 0xc4200bbe00, 0xc4203f74d0, 0x22fc818, 0xc4202bac90, 0x0, 0x0)
	/home/crazykev/go-project/src/k8s.io/frakti/vendor/google.golang.org/grpc/server.go:608 +0xc50
k8s.io/frakti/vendor/google.golang.org/grpc.(*Server).handleStream(0xc420418f00, 0x2322ee0, 0xc420490a20, 0xc4200bbe00, 0xc4202bac90)
	/home/crazykev/go-project/src/k8s.io/frakti/vendor/google.golang.org/grpc/server.go:766 +0x6b0
k8s.io/frakti/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc4204bbab0, 0xc420418f00, 0x2322ee0, 0xc420490a20, 0xc4200bbe00)
	/home/crazykev/go-project/src/k8s.io/frakti/vendor/google.golang.org/grpc/server.go:419 +0xab
created by k8s.io/frakti/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
	/home/crazykev/go-project/src/k8s.io/frakti/vendor/google.golang.org/grpc/server.go:420 +0xa3
```

**Comment by Crazykev** [comment]
By the way, I have no cni plugin configured.
```
E0315 20:22:15.130748   14017 ocicni.go:129] error updating cni config: No networks found in /etc/cni/net.d
W0315 20:22:15.131237   14017 ocicni.go:55] Error in finding usable CNI plugin - No networks found in /etc/cni/net.d
```

**Comment by resouer** [comment]
fixed by #92
